ui/src/app/machines/views/MachineDetails/MachineNetwork/BondForm/BondForm.tsx

```diff
@@ -80,40 +211,90 @@ const BondForm = ({
   const rows = editingMembers
     ? validNics.map(({ id, links }) => ({ nicId: id, linkId: links[0]?.id }))
     : selected;
-  let editTooltip: string | null = null;
-  let editDisabled = false;
-  if (!editingMembers && validNics.length === 2) {
-    // Disable the button to add more members if there are no more to choose
-    // from.
-    editTooltip = "There are no additional valid members";
-    editDisabled = true;
-  } else if (editingMembers && selected.length < 2) {
-    // Don't let the user update the selection if they haven't chosen at least
-    // two interfaces.
-    editTooltip = "At least two interfaces must be selected";
-    editDisabled = true;
-  }
 
   return (
     <FormCard sidebar={false} stacked title="Create bond">
-      <InterfaceFormTable
-        editPrimary
-        interfaces={rows}
-        selected={selected}
-        selectedEditable={editingMembers}
-        setSelected={setSelected}
-        systemId={systemId}
-      />
-      <Tooltip message={editTooltip}>
-        <Button
-          data-test="edit-members"
-          disabled={editDisabled}
-          onClick={() => setEditingMembers(!editingMembers)}
-        >
-          {editingMembers ? "Update bond members" : "Edit bond members"}
-        </Button>
-      </Tooltip>
-      <button onClick={close}>Cancel</button>
+      <FormikForm
+        allowUnchanged
+        buttons={FormCardButtons}
+        cleanup={cleanup}
+        errors={errors}
+        initialValues={{
+          ...networkFieldsInitialValues,
+          bond_downdelay: 0,
+          bond_lacp_rate: "",
+          bond_mode: BondMode.ACTIVE_BACKUP,
+          bond_miimon: 0,
+          bond_updelay: 0,
+          bond_xmit_hash_policy: "",
+          fabric: vlan?.fabric || "",
+          linkMonitoring: "",
+          mac_address: firstNic?.mac_address || "",
+          name: nextName,
+          primary: firstNic?.id.toString(),
```

Comment:
How come this needs to be converted to a string?

Reply:
The values for the primary radio buttons get converted to strings (every value is a string in an input) so we need to convert this to a string also so that the values match to show the currently selected option.

Formik does seem to eventually convert it's values back to numbers, but it seems a bit wonky.